reporting_app/templates/proc_report.html

```diff
@@ -1,91 +1,75 @@
+{% macro proc_header() -%}
+<script src="//cdnjs.cloudflare.com/ajax/libs/vis/4.19.1/vis.min.js"></script>
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/vis/4.19.1/vis.min.css" type="text/css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
+<script src="/static/stage_timeline.js"></script>
+
+<style>
+    .vis-item .vis-item-overflow {
+        overflow: visible;
+    }
+
+    div.vis-tooltip {
+        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;  // override vis defaults back to bootstrap
+        background-color: #f9f9f9;
+    }
+</style>
+{%- endmacro %}
+
 
 {% macro proc_timeline(proc) -%}
 {# args:
     proc (dict) - analysis_driver_proc from the Rest API.
 #}
     <br/>
-    <ul>
-        <li>Started: {{ proc._created }}</li>
-        <li>Ended:
+    <div>
+        <div>Started: {{ proc._created }}</div>
+        <div>
             {% if proc.end_date %}
-            {{ proc.end_date }}
+            Ended: {{ proc.end_date }}
             {% else %}
-            running with pid: {{ proc.pid }}
+            Running with pid: {{ proc.pid }}
             {% endif %}
-        </li>
-        <li>
+        </div>
+        <div>
             Status:
             {% if proc.status %}
             {{ proc.status }}
             {% else %}
             inactive
             {% endif %}
-        </li>
-    </ul>
+        </div>
+    </div>
     <br/>
 
-    <div id="{{ proc.proc_id }}"></div>
+    <div id="timeline_{{ proc.proc_id }}"></div>
 
     <script type="text/javascript">
-        var container = document.getElementById('{{ proc.proc_id }}');
-        var stages = {{ proc.stages|list|safe }};
-        var stage_names = stages.map(function(p){
-            if (p.date_finished){
-                return {
-                    content: p.stage_name,
-                    'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
-                    'end': moment(p.date_finished, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS')
-                }
-            } else {
-                return {
-                    'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
-                    'style': "color: red; background-color: pink;"
-                }
-            }
-        });
-
-
-        var items = new vis.DataSet(stage_names);
-        startDate = (moment('{{ proc._created }}', "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'))
-        startMargin = moment(startDate).subtract(1, 'days')
-        endMargin = moment(startDate).add(6, 'days')
-        var options = {
-            min: startMargin,
-            max: endMargin
-        }
-        var timeline = new vis.Timeline(container, items, options);
+        stage_timeline("timeline_{{ proc.proc_id }}", {{ proc|safe }});
```

Comment:
It seems to work with just `safe` but I would pass it through `tojson` as well to make sure that the conversion is properly done